azure-pipelines.yml

```diff
@@ -324,83 +324,6 @@ jobs:
     inputs:
       filePath: build_scripts\windows\scripts\test_zip_installation.ps1
 
-- job: BuildDockerImageAlpine
-  continueOnError: true
-  displayName: Build Docker Image Alpine
-  strategy:
-    matrix:
-      ${{ each arch in parameters.architectures }}:
-        ${{ arch.name }}:
-          pool: ${{ arch.pool }}
-          artifactName: docker-${{ arch.value }}
-          dockerfile: alpine.dockerfile
```

Comment:
Should we remove `alpine.dockerfile` too?

Reply:
I don't remove it because `centos7.dockerfile` and `fedora.dockerfile` are kept.
I prefer to remove it as it can be restored in git.

Reply:
Then let's do it.